Use new domains illustrations

delputnam opened the pull request:
#### Changes proposed in this Pull Request

We have some new illustrations to use for the domains pages. This PR just replaces some of the existing illustrations with these new ones.

#### Testing instructions

For sites with a plan, but no domain, make sure that you see the new `free-domain.svg` in the domain list:

<img width="1085" alt="Screen Shot 2021-12-07 at 1 19 54 PM" src="https://user-images.githubusercontent.com/1379730/145090987-5dbf6b1c-00e6-4daf-8360-9e49575cee50.png">

<img width="1085" alt="Screen Shot 2021-12-07 at 1 20 57 PM" src="https://user-images.githubusercontent.com/1379730/145091004-fa182fa7-0c08-4b84-9653-8158a0964990.png">

<img width="370" alt="Screen Shot 2021-12-07 at 1 21 26 PM" src="https://user-images.githubusercontent.com/1379730/145091013-f5cf1911-bd08-4284-9229-3c66b6922941.png">

Go through the connect a domain flow and on the last page, make sure you see the new "connect" illustration:

<img width="1103" alt="Screen Shot 2021-12-07 at 1 23 59 PM" src="https://user-images.githubusercontent.com/1379730/145091191-011f5246-d276-4fd6-9b08-2056816251c8.png">

On the "Use a domain I own" flow, make sure that the input page shows the new "domain" illustration:

<img width="1080" alt="Screen Shot 2021-12-07 at 1 38 36 PM" src="https://user-images.githubusercontent.com/1379730/145091418-e8093e2f-7368-4ec3-b70f-c87a3a8b9c5b.png">

...and that the next page shows the new transfer and new connection illustrations:

<img width="1078" alt="Screen Shot 2021-12-07 at 12 56 41 PM" src="https://user-images.githubusercontent.com/1379730/145091536-641a7622-c0b7-422e-a45c-b9d53a9b3fbd.png">

Checkout pages:

<img width="1163" alt="Screen Shot 2021-12-07 at 2 19 14 PM" src="https://user-images.githubusercontent.com/1379730/145092635-d3e8f585-7806-4edd-a7ab-2b468ac832c4.png">

<img width="1163" alt="Screen Shot 2021-12-07 at 2 20 30 PM" src="https://user-images.githubusercontent.com/1379730/145092660-e2134d18-501f-4e88-8cbd-a0fdf754e57b.png">

<img width="1163" alt="Screen Shot 2021-12-07 at 2 22 48 PM" src="https://user-images.githubusercontent.com/1379730/145092920-bfd95c0f-3f38-4238-9224-77eaa61cc6de.png">

delputnam committed:
Use new domains illustrations

poligilad-auto commented:
Thanks for updating this @delputnam!
Looking good to me. 

**2 things about the checkout pages**:
1. The illustrations seem a bit big. Did you enlarge them in some way? They should be the same size as the the "empty state" illustration when a user has no domains.
2. They do seem a bit strange on the blue background. There is a [recent PR](https://github.com/Automattic/wp-calypso/pull/58668) that's updating the style of that page for the Marketplace checkout page. I created [this PR](https://github.com/Automattic/wp-calypso/issues/58895) hoping that some can pickup updating the style of the page for all instances. Do you know who could potentially take ownership of that?

delputnam committed:
Update transfer and registration checkout pages

delputnam committed:
Adjust sizes and add more uses of the  illustration

delputnam commented:
> * The illustrations seem a bit big. Did you enlarge them in some way? They should be the same size as the the "empty state" illustration when a user has no domains.

The illustrations on the thank you pages had their sizes set based on the height. Since the new illustrations have a different aspect ratio, the widths were wider than the previous illustrations. So, I've updated the thank you pages to set the sizes based on the width.

<img width="1090" alt="Screen Shot 2021-12-09 at 8 42 44 AM" src="https://user-images.githubusercontent.com/1379730/145442478-3defc44a-a7b2-42cf-ae81-d76878af45f7.png">
<img width="1090" alt="Screen Shot 2021-12-09 at 8 42 48 AM" src="https://user-images.githubusercontent.com/1379730/145442480-6968b544-091e-4591-8365-d1ecb1765601.png">
<img width="1090" alt="Screen Shot 2021-12-09 at 8 42 51 AM" src="https://user-images.githubusercontent.com/1379730/145442481-fca82e04-ec16-4d4a-8d41-2c7da0fed2d3.png">


I also added the new "domain" illustration to the "empty emails" page as we discussed here: p1639056145192500/1639055655.192400-slack-C020919TDRA

<img width="1090" alt="Screen Shot 2021-12-09 at 12 03 42 PM" src="https://user-images.githubusercontent.com/1379730/145442552-4d2c198b-4b34-45e9-a2e6-fc2c93c7557a.png">

And to the "domain-only" page:
<img width="1090" alt="Screen Shot 2021-12-09 at 11 54 52 AM" src="https://user-images.githubusercontent.com/1379730/145442635-7ee84d58-1ecb-4367-8047-470d5784b736.png">

And the "empty" domain search page:
<img width="1090" alt="Screen Shot 2021-12-09 at 11 52 34 AM" src="https://user-images.githubusercontent.com/1379730/145442725-e1a9db8a-2522-4d24-b3ff-9b63c581cd92.png">

I think that's everything.

delputnam commented on client/components/domains/register-domain-step/index.jsx:
```diff
@@ -648,7 +648,7 @@ class RegisterDomainStep extends Component {
 					className="register-domain-step__placeholder"
 					illustration={ Illustration }
-					illustrationWidth={ 180 }
+					illustrationWidth={ 280 }
```
Setting the width to 280 here makes the height of the new illustration about the same as the former illustration.

delputnam commented on client/components/domains/register-domain-step/style.scss:
```diff
@@ -153,6 +153,10 @@
 }
 
+.register-domain-step__placeholder.empty-content {
+	padding-bottom: 30px;
```
This evens out the whitespace in the content area above and below the new illustration.

delputnam commented on client/components/thank-you/index.tsx:
```diff
@@ -177,8 +177,8 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
 		img {
-			width: auto;
-			height: 200px;
+			width: ${ thankYouImage.width ? null : 'auto' };
+			height: ${ thankYouImage.height ? null : '200px' };
```
This allows us to pass in custom width and height values that get set on the `<img>` element as necessary since the aspect ratio of the new illustrations are different.

leonardost reviewed (approved):
Everything looks great! I think I've checked all the places where the illustrations should have been updated and everything LGTM 👍

poligilad-auto commented:
@delputnam the illustrations on the email page & the domain only page seems big as well. Can we make it 220px width?
